CMakeLists.txt

```diff
@@ -153,7 +183,19 @@ if(AVIF_LOCAL_LIBSHARPYUV)
         set(LIBSHARPYUV_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/ext/libwebp" PARENT_SCOPE)
         set(LIBSHARPYUV_LIBRARY ${LIB_FILENAME} PARENT_SCOPE)
     endif()
+else(AVIF_LOCAL_LIBSHARPYUV)
+    find_package(libsharpyuv QUIET) # not required
+    if(libsharpyuv_FOUND)
+        message(STATUS "libavif: libsharpyuv found; sharp rgb to yuv conversion enabled.")
```

Comment:
The `if(AVIF_LOCAL_LIBSHARPYUV)` branch should also emit a message indicating sharp rgb to yuv conversion is enabled.

Reply:
Done